src/modules/viewmodel.rs

```diff
@@ -0,0 +1,40 @@
+//! `bxt_viewmodel`
+
+use super::Module;
+use crate::hooks::engine;
+use crate::modules::cvars::CVar;
+use crate::utils::*;
+
+pub struct Viewmodel;
+impl Module for Viewmodel {
+    fn name(&self) -> &'static str {
+        "Viewmodel stuffs."
+    }
+
+    fn description(&self) -> &'static str {
+        "Do viewmodel stuffs."
```

Comment:
```suggestion
        "Removing the viewmodel."
```
Please write proper strings next time, they show up on the wiki and in `bxt_help`. Other modules have examples.